samples/docker/README.md

```diff
@@ -4,6 +4,48 @@
 
 The following instructions detail how to build and run the FHIR Server in Docker on Linux.
 
+## Use CI image
+
+If it is not desirable to clone this repository and build locally an image of the most recent CI build is available from the Azure Container Repository HealthPlatformRegistry. 
+
+Using docker-compose this image can be started with the following steps:
+1. Open a terminal window.
+2. Login to the Azure Container Registry 'healthplatformregistry' (https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-docker-cli)
+3. Set the enviornment variable SAPASSWORD to what you want the SQL access password to be. Be sure to follow the SQL password complexity rules: https://docs.microsoft.com/en-us/sql/relational-databases/security/password-policy?view=sql-server-ver15#password-complexity.
+4. Copy & save a local version of the docker-compose file from the release directory of the fhir-server project.
+5. Run the command: 
+
+```bash
+docker-compose up -d .
+```
+
+6. After giving the container a minute to start up it should be accessable at http://localhost/metadata.
+
+
+Using just docker this image can be started with the following steps. Replace the `<SA_PASSWORD>` below with your chosen SQL connection password, following the complexity rules linked above.
+1. Open a terminal window.
+2. Login to the Azure Container Registry 'healthplatformregistry' (https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-docker-cli).
+3. Run the command: 
+
+```bash
+docker network create fhir_network
+```
+
+4. Run the command: 
+
+```bash
+docker run --net fhir_network --name fhir_sql -e SA_PASSWORD=<SA_PASSWORD> -e ACCEPT_EULA="Y" -d mcr.microsoft.com/mssql/server
+```
+
+5. Give the SQL server a minute to start up.
+6. Run the command: 
+
+```bash
+docker run --net fhir_network -e FhirServer__Security__Enabled="false" -e SqlServer__ConnectionString="Server=tcp:fhir_sql,1433;Initial Catalog=FHIR;Persist Security Info=False;User ID=sa;Password=<SA_PASSWORD>;MultipleActiveResultSets=False;Connection Timeout=30;" -e SqlServer__AllowDatabaseCreation="true" -e SqlServer__Initialize="true" -e DataStore="SqlServer" -p 80:80 -d healthplatformregistry.azurecr.io/fhirserverR4_fhir-api azure-fhir-api
```

Comment:
does it provide persistence? if no, should we add steps here?

Reply:
Added note that persistence is not provided with this deployment method.